tests/atomic/pull_digest.yml

```diff
@@ -0,0 +1,34 @@
+---
+# set ft=ansible
+#
+
+- name: Set image name
+  set_fact:
+    fq_name: "docker.io/busybox"
+    digest:
```

Comment:
These two facts should probably be prefixed to make them unique to this playbook.